qiskit_nature/properties/second_quantization/electronic/electronic_energy.py

```diff
@@ -101,11 +57,65 @@ def __init__(
             reference_energy: an optional reference energy (such as the HF energy).
             energy_shift: an optional dictionary of energy shifts.
         """
-        super().__init__(self.__class__.__name__, basis, electronic_integrals, shift=energy_shift)
+        super().__init__(self.__class__.__name__, electronic_integrals, shift=energy_shift)
+        self._nuclear_repulsion_energy = nuclear_repulsion_energy
         self._reference_energy = reference_energy
 
+        # Additional, purely information data (i.e. currently not used by the Stack itself).
+        self._orbital_enerfies: np.ndarray = None
+        self._kinetic: ElectronicIntegrals = None
+        self._overlap: ElectronicIntegrals = None
+        self._fock: ElectronicIntegrals = None
+
+    @property
+    def orbital_energies(self) -> np.ndarray:
+        """Returns the orbital energies.
+
+        If no spin-distinction is made, this is a 1-D array, otherwise it is a 2-D array.
+        """
+        return self._orbital_energies
+
+    @orbital_energies.setter
+    def orbital_energies(self, orbital_energies: np.ndarray) -> None:
+        """Sets the orbital energies."""
+        self._orbital_energies = orbital_energies
+
+    @property
+    def kinetic(self) -> ElectronicIntegrals:
+        """Returns the AO kinetic integrals."""
+        return self._kinetic
+
+    @kinetic.setter
+    def kinetic(self, kinetic: ElectronicIntegrals) -> None:
+        """Sets the AO kinetic integrals."""
+        self._kinetic = kinetic
+
+    @property
+    def overlap(self) -> ElectronicIntegrals:
+        """Returns the AO overlap integrals."""
+        return self._overlap
+
+    @overlap.setter
+    def overlap(self, overlap: ElectronicIntegrals) -> None:
+        """Sets the AO overlap integrals."""
+        self._overlap = overlap
+
+    @property
+    def fock(self) -> ElectronicIntegrals:
+        """Returns the AO Fock operator.
+
+        In the case of DFT this cannot be reconstructed from the 1- and
+        2-electron integrals because of the XCF-dependent term.
+        """
+        return self._fock
+
+    @fock.setter
+    def fock(self, fock: ElectronicIntegrals) -> None:
+        """Returns the AO Fock operator."""
```

Comment:
```suggestion
        """Sets the AO Fock operator."""
```

Reply:
As mentioned above I have removed the (so far unused) `fock` attribute for now.